spring-boot-app-sso-auth-code-flow/src/main/java/com/vmware/tap/fragments/appssoauthcodeflow/web/OidcUserController.java

```diff
@@ -0,0 +1,17 @@
+package com.vmware.tap.fragments.appssoauthcodeflow.web;
+
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.security.oauth2.core.oidc.user.OidcUser;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class OidcUserController {
+
+    @GetMapping("/protected/profile")
```

Comment:
The `WebSecurityConfiguration.java` file uses tabs and this one spaces for indents, should be consistent

Reply:
Done